### PR TITLE
Adds a Scottish Trait for non-dwarves.

### PR DESCRIPTION
**Pull request by P-MCT**
## About the PR

Adds a Scottish trait that's 1-1 with a dwarven accent. 

Based/Ported from a PR on https://github.com/DeltaV-Station/Delta-v/pull/608

## Why / Balance

Unlocks the accent trait to other races. (FREEEEEDOOOOOOOM!!!!!!)

Might consider expanding or making a seperate PR to allow the dwarves to be accent-less.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

https://github.com/user-attachments/assets/073a7ea6-396c-4233-adf8-6b8593660016

**Changelog**

Added: Scottish trait/accent.

### CODE REVIEW

File: Content.Server/_RMC14/Speech/Components/ScottishAccentComponent.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+using Content.Server.RMC14.Speech.EntitySystems;
+
+namespace Content.Server.RMC14.Speech.Components;
+
+[RegisterComponent]
+[Access(typeof(ScottishAccentSystem))]
+public sealed partial class ScottishAccentComponent : Component
+{ }
```
Comment (Dutch-VanDerLinde) [inline Content.Server/_RMC14/Speech/Components/ScottishAccentComponent.cs:8]:
```suggestion
```

File: Content.Server/_RMC14/Speech/EntitySystems/ScottishAccentsSystem.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        msg = _replacement.ApplyReplacements(msg, "scottish");
+
+        return msg;
+    }
+
+    private void OnAccentGet(EntityUid uid, ScottishAccentComponent component, AccentGetEvent args)
+    {
+        args.Message = Accentuate(args.Message, component);
+    }
+}
```
Comment (Dutch-VanDerLinde) [inline Content.Server/_RMC14/Speech/EntitySystems/ScottishAccentsSystem.cs:33]:
```suggestion
```

File: Resources/Locale/en-US/_RMC14/accent/scottish.ftl [hunk 3]
```diff
@@ -0,0 +1,469 @@
...
+accent-scottish-words-replace-160 = snake
+
+accent-scottish-words-161 = syndicates
+accent-scottish-words-replace-161 = snakes
+
+accent-scottish-words-162 = nukies
+accent-scottish-words-replace-162 = reddies
+
+accent-scottish-words-163 = syndicate agents
+accent-scottish-words-replace-163 = snakey bastards
```
Comment (Dutch-VanDerLinde) [inline Resources/Locale/en-US/_RMC14/accent/scottish.ftl:469]:
```suggestion
```

File: Resources/Prototypes/_RMC14/Accents/word_replacements.yml [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    accent-scottish-words-154: accent-scottish-words-replace-154
+    accent-scottish-words-155: accent-scottish-words-replace-155
+    accent-scottish-words-156: accent-scottish-words-replace-156
+    accent-scottish-words-157: accent-scottish-words-replace-157
+    accent-scottish-words-158: accent-scottish-words-replace-158
+    accent-scottish-words-159: accent-scottish-words-replace-159
+    accent-scottish-words-160: accent-scottish-words-replace-160
+    accent-scottish-words-161: accent-scottish-words-replace-161
+    accent-scottish-words-162: accent-scottish-words-replace-162
+    accent-scottish-words-163: accent-scottish-words-replace-163
```
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/_RMC14/Accents/word_replacements.yml:166]:
```suggestion
```

File: Resources/Prototypes/_RMC14/Traits/cm_neutral.yml [hunk 5]
```diff
@@ -51,3 +51,10 @@
   description: trait-southern-desc
   components:
   - type: SouthernAccent
+
+  - type: trait
+  id: ScottishAccent
+  name: trait-scottish-accent-name
+  description: trait-scottish-accent-desc
+  components:
+    - type: ScottishAccent
```
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/_RMC14/Traits/cm_neutral.yml:60]:
```suggestion
    - type: ReplacementAccent
      accent: dwarf
```

**Review by Dutch-VanDerLinde — CHANGES_REQUESTED** [review]
you basically recreated the entire dwarf accent just with a new name
delete all the scottish code
(ScottishAccentComponent, ScottishAccentsSystem, scottish.ftl, word_replacements.ftl)
and use the already existing ReplacementAccent component with dwarf

File: Resources/Prototypes/_RMC14/Traits/cm_neutral.yml [hunk 6]
```diff
@@ -51,3 +51,11 @@
   description: trait-southern-desc
   components:
   - type: SouthernAccent
+
+  - type: trait
```
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/_RMC14/Traits/cm_neutral.yml:55]:
```suggestion
- type: trait
```

**Review by Dutch-VanDerLinde — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_RMC14/Traits/cm_neutral.yml [hunk 7]
```diff
@@ -51,3 +51,11 @@
...
   components:
   - type: SouthernAccent
+
+- type: trait
+  id: ScottishAccent
+  name: trait-scottish-accent-name
+  description: trait-scottish-accent-desc
+  components:
+    - type: ReplacementAccent
+      accent: dwarf
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Traits/cm_neutral.yml:61]: By convention we put sequence nodes (the dash) on the same level as the node before (components)
```suggestion
  - type: ReplacementAccent
    accent: dwarf
```

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!
There were a few leftover empty files but I deleted those now

**Pull request merged**